_config.yml

```diff
@@ -16,11 +16,9 @@ logo: assets/images/blog-icon.png
 logo_dark: assets/images/favicon.png
 favicon: assets/images/favicon.png
 
-# Social info
-navigation: True # show the navbar links
-subscribers: True # you need to connect an external service for this to work,
-# if 'True' the submit button is disabled for now, but you can change that
-# by editing `_includes/subscribe-form.html`
+#Need add your subscribe mailchimp link list
+mailchimp-list: //redgadgets.us10.list-manage.com/subscribe/post?u=210acce5db69d3d4a04b0e25d&amp;id=08c6708f40
```

Comment:
Should be "https://wott.us20.list-manage.com/subscribe/post?u=250c99a25b42ad842add5bf41&amp;id=ac12838d30"